src/pytorch_metric_learning/losses/supcon_loss.py

```diff
@@ -0,0 +1,44 @@
+from ..distances import CosineSimilarity
+from ..reducers import AvgNonZeroReducer
+from ..utils import common_functions as c_f
+from ..utils import loss_and_miner_utils as lmu
+from .generic_pair_loss import GenericPairLoss
+
+
+class SupConLoss(GenericPairLoss):
+    def __init__(self, temperature=0.1, **kwargs):
```

Comment:
I removed the `base_temperature` parameter from https://github.com/HobbitLong/SupContrast/blob/master/losses.py

This is not mentioned in the paper, and it's useless if it's set to be equal to temperature.